shared/common-adapters/button.tsx

```diff
@@ -27,12 +27,10 @@ export type ButtonColor = 'blue' | 'red' | 'green' | 'purple' | 'black' | 'yello
 type WithIconProps =
   | {
       icon?: never
-      tooltip?: string
     }
   | {
       icon: IconType
       iconColor?: Styles.Color
-      tooltip: string
```

Comment:
Perhaps go through and delete the places where we used `tooltip=''` to get around this if we're removing this rule?

Do you know why it was there in the first place? a11y? Something else?

Reply:
Done :+1: not sure why it was required.